easybuild/easyconfigs/s/Szip/Szip-2.1.1-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Szip'
+version = '2.1.1'
+
+homepage = 'http://www.hdfgroup.org/doc_resource/SZIP/'
+description = "Szip compression software, providing lossless compression of scientific data"
+
+toolchain = {'name': 'foss', 'version': '2017a'}
```

Comment:
Ok, I discarded the foss/2017a file and "cloned" the GCCcore/6.4 into a GCCcore/6.3.  To be submitted later.